AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Periodic_GPU.py

```diff
@@ -0,0 +1,101 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+import logging
+import os
+import psutil
```

Comment:
The `psutil` import isn't used and removing it, would allow these python scripts to work in debug configuration.
It was recently removed from the o3de CLI utils.py to allow the a debug configuration Project Manager to be used

Reply:
I've removed the import